modes/settings_menues.h

```diff
@@ -74,42 +74,57 @@ class ShowLengthStyle {
 };
 
 template<class SPEC>
-struct ChangeBladeLengthBlade1 : public SPEC::MenuBase {
+struct ChangeBladeLengthMode : public SPEC::MenuBase {
   virtual int blade() { return menu_current_blade; }
   void mode_activate(bool onreturn) override {
     int len = prop_GetBladeLength(blade());
     int maxlen = prop_GetMaxBladeLength(blade());
-    saved_len_ = len;
     if (len == -1) len = maxlen;
-    this->pos_ = len;
+    saved_len_ = len;
```

Comment:
Moving it here saves it at maxlen as opposed to -1 if it was.
Does this make sense?

Reply:
Roger roger.